components/playground/instance/process.go

```diff
@@ -50,8 +57,9 @@ func (p *process) Pid() int {
 // Uptime implements Instance interface.
 func (p *process) Uptime() string {
 	s := p.cmd.ProcessState
-	if s != nil && s.Exited() {
-		return "exited"
+
+	if s != nil {
```

Comment:
Can we retrieve the `ProcessState` if the current process is running? If so this method won't return the real `Uptime`.

Reply:
131     // ProcessState contains information about an exited process,
132     // available after a call to Wait or Run.
133     ProcessState *os.ProcessState

It is only available after the proccss exited and we call Wait on the Cmd.